default/scripting/species/SP_LAENFA.focs.txt

```diff
@@ -6,7 +6,7 @@ Species
     CanProduceShips
     CanColonize
 
-    tags = [ "PHOTOTROPHIC" "TELEPATHIC" "GOOD_POPULATION" "AVERAGE_SUPPLY" "GREAT_DETECTION" "GREAT_STEALTH" "BAD_ATTACKTROOPS" "PEDIA_PC_FUEL" "PEDIA_PHOTOTROPHIC_SPECIES_CLASS" "INDEPENDENT_HAPPINESS" ]
```

Comment:
The fuel pedia label was seemingly there due to their fuel regeneration effect... Does the label just not work in practice despite that?

Reply:
It was added by stpa to laenfa very recently but it categorizes sly as a fuel part because of \encyclopedia\shipparts\pc_fuel.foc
![PC_FUEL](https://user-images.githubusercontent.com/43386669/154351399-5a6011e7-f6a4-4102-bb81-802f33a93e16.png)
s
It was presumably copied from the entry in sly
![Sly](https://user-images.githubusercontent.com/43386669/154351487-69662876-e671-4d47-b620-254fd66fe46d.png)

I have no idea why its on sly without some serious digging

Reply:
At least for Laenfa, I think that's the idea... they effectively act as a fuel part.

Reply:
I guess leave it there and ask on the forums...?

Reply:
Removed this commit. Its easy enough to change later anway